### ISSUE TITLE
Add required performance_schema instruments

### ISSUE THREAD

**Issue by gkristic** [VividCortex/docs#244]
In order to collect InnoDB mutex metrics, we need the `wait/sync/mutex/innodb/%` instruments enabled in performance_schema. This is currently not mentioned in the docs. For those to be enabled, the following line has to be added to a my.cnf:

```
performance-schema-instrument='wait/synch/mutex/innodb/%=ON'
```

Instruments can be enabled/disabled at run-time, so there's no need to ask customers to restart their instances for this reason. Here's how to update a running MySQL:

```sql
UPDATE performance_schema.setup_instruments SET ENABLED='YES' WHERE NAME LIKE 'wait/synch/mutex/innodb/%';
```

Likewise, documents are missing the requirement for the `wait/io/socket/sql/%` instruments, that helps identifying connections to aid the sniffer. This one is not required when monitoring in an off-host configuration, given that the sniffer does not run. Enabling these instruments goes just like for `wait/sync/mutex/innodb/%`, and can also be done online with an UPDATE statement.

**Comment by gkristic** [comment]
We should make it clear that enabling these instruments may be expensive.

**Comment by seaparks** [comment]
@gkristic Agreed - Thor and I met yesterday re: rewriting our setup docs to make clear all of the required information (install, permissions, stuff like this) and it'll get incorporated soon!

**Comment by efepapa** [comment]
[on-host and off-host]

Populate *PROFILER | InnoDB Mutexes*. **expensive**; requires vc-mysql-metrics restart:

[my.cnf] `performance-schema-instrument='wait/synch/mutex/innodb/%=ON'`

[runtime] `UPDATE performance_schema.setup_instruments SET ENABLED='YES',TIMED='YES' WHERE NAME LIKE 'wait/synch/mutex/innodb/%';`

---

[on-host only]

Due to packet-loss or long-lived/persistent connections, our sniffing agent sometimes fails to catch a client/server handshake and so misses user / starting database.  E.g. metrics and samples for `unknown_user` or `unknown_database`.

There's a mitigation feature and it has very low overhead: enable connection tracking in vc-mysql-metrics to aid vc-mysql-query identify the correct user/db/etc. Requires vc-mysql-metrics restart:

[my.cnf] `performance_schema_instrument='wait/io/socket/sql/%=counted'`

[runtime] `UPDATE performance_schema.setup_instruments SET ENABLED='YES' WHERE NAME LIKE 'wait/io/socket/sql/%';`
